docs/reference/glossary.md

```diff
@@ -264,14 +285,23 @@ containers.
 
 ## Toolbox
 
-Docker Toolbox is the installer for Mac and Windows users.
+[Docker Toolbox](https://docs.docker.com/toolbox/overview/) is a legacy
+installer for Mac and Windows users. It uses Oracle VirtualBox for
+virtualization.
+
+For Macs running OS X El Capitan 10.11 and newer macOS releases, [Docker for
+Mac](https://docs.docker.com/docker-for-mac/) is the better solution.
+
+For Windows 10 systems that support Microsoft Hyper-V (Professional, Enterprise
+and Education), [Docker for
+Windows](https://docs.docker.com/docker-for-windows/) is the better solution.
 
 
 ## Union file system
 
-Union file systems, or UnionFS, are file systems that operate by creating layers, making them
-very lightweight and fast. Docker uses union file systems to provide the building
-blocks for containers.
+Union file systems, or UnionFS, are file systems that operate by creating
+layers, making them very lightweight and fast. Docker uses union file systems to
```

Comment:
Wonder if we should leave out `UnionFS`, as it's an _implementation_ of a union filesystem. (https://en.m.wikipedia.org/wiki/UnionFS), same as aufs, overlay

Should we have a section on "Copy on Write"?